build/AzurePipelinesTemplates/WindowsAppSDK-CallTouchDownBuild-Steps.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+- name: ResourceFilePath
+  type: string
+  default: ""
+- name: GitAction
+  type: string
+  default: 'COMMIT'
+  values:
+  - 'None'
+  - 'COMMIT'
+
+steps:
+- checkout: self
+  persistCredentials: true
+
+# This is needed to avoid credentials errors for later in the TDBuild Task
+- script: |
+    git config --global user.email "you@example.com"
```

Comment:
Can we put something in here to indicate the commit is coming from the touchdown build?